app/controllers/v1/applications_controller.rb

```diff
@@ -2,6 +2,11 @@ class V1::ApplicationsController < ApplicationController
   def create
     application = LegalAidApplication.new
 
+    if params[:proceeding_type_codes]
```

Comment:
I suspect the changes in this file might be better placed in the model where a validation can run if any of the passed in IDs do not already exist - does anyone else have an opinion on this?